unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsPanelHUD/QuestsPanelPopup.cs

```diff
@@ -38,8 +38,8 @@ private void Awake()
             closeButton.onClick.AddListener(Close);
 
             pinQuestToggle.onValueChanged.AddListener(OnPinToggleValueChanged);
-            baseCollection.OnAdded += OnPinnedQuests;
-            baseCollection.OnRemoved += OnUnpinnedQuest;
+            pinnedQuests.OnAdded += OnPinnedQuests;
+            pinnedQuests.OnRemoved += OnUnpinnedQuest;
```

Comment:
These 2 should unsubscribe in `OnDestroy()` right? (It's not something introduced in this PR but still)

Reply:
Nice catch, thx!